### PR TITLE
PullRequest for Issue1297: seperate CLS specific files from the acquamanCommon.pri

### PR DESCRIPTION
**Pull request by XuetaoLiu-CLS**
https://github.com/acquaman/acquaman/issues/1297

- add the `clsCompositeCommon` folder and `CLSAquamanCommon.pri` file.
- changed the includes for `acquamanCommon.pri` in CLS acquaman project
files

@davidChevrier @dretrex @helfrij @iainworkman @epengr @dleacock

### CODE REVIEW

**Comment by davidChevrier** [comment]
Hey @XuetaoLiu-CLS 
Overall, this is good. I think there are a few consistency issues that should be address.
First, this isn't a "compositeCommon" approach (CLS or otherwise). We're just trying to aggregate all of the CLS classes in one place. As such, I would suggest just placing the file in the acquaman directory beside the existing acquamanCommon.pri file.
Second, since it's not a composite approach, I think the file should really take on a name similar to the acquamanCommon.pri file ... problably clsCommon.pri
When the changes are made I'll do a pull and a clean build to make sure everything builds from scratch.

File: BioXASImagingAcquaman_internal.pro [hunk 1]
```diff
@@ -1,4 +1,4 @@
-include ( acquamanCommon.pri )
+include ( $$PATH_TO_AM/clsCompositeCommon/CLSAcquamanCommon.pri )
```
Comment (dretrex) [inline BioXASImagingAcquaman_internal.pro]: You shouldn't need the ```$$PATH_TO_AM/``` in these include calls.

File: clsCompositeCommon/CLSAcquamanCommon.pri [hunk 2]
```diff
@@ -0,0 +1,93 @@
+
+DESTDIR = build
```
Comment (dretrex) [inline clsCompositeCommon/CLSAcquamanCommon.pri]: I understand the desire to move the CLS common stuff into the same place.  But I don't like having the beamline specific pieces in a pri file that specifically designates CLS.  It's like finding an IDEAS class in the VESPERS pri file.  If we are going to split them up, then it needs to be more thorough.

**Comment by dretrex** [comment]
Unrelated, but @XuetaoLiu-CLS, could you add your other email to your list of emails in your github account so your contributions are properly mapped to your account?

**Comment by XuetaoLiu-CLS** [comment]
@davidChevrier file is renamed and moved to the acquaman folder
@dretrex yeah, it is the username configuration stuff. I thought I was using my gmail :(. It is fixed now.

**Comment by davidChevrier** [comment]
Hey @XuetaoLiu-CLS @dretrex 
I see there's still some SGM, VESPERS, and REIXS stuff caught up in the clsCommon.pri file ... do we want a resolution on that? Otherwise this looks good and builds on my end. I can do the merge with master on my side and push it up and merge the pull request when ready.

**Comment by dretrex** [comment]
Those are all the old file writers I think.  I'd really like to do something about them since we exclusively use our ascii and CDF file loaders now.

**Comment by dretrex** [comment]
But not in this issue.

**Comment by XuetaoLiu-CLS** [comment]
@davidChevrier Yes, I talked with @dretrex about this yesterday. There is more to do to solve that.

As you might also notice that there is one SGM filderLoader left in the acquamanCommon.pri :(

**Comment by XuetaoLiu-CLS** [comment]
@davidChevrier @dretrex the fileLoaders are moved to clsFileLoader.pri

**Comment by davidChevrier** [comment]
@XuetaoLiu-CLS I'm getting a build error:
/acquaman/source/ui/CLS/CLSSIS3820ScalerView.cpp:27: error: 'ui/CLS/CLSSIS3820ScalerChannelViewWithDarkCurrent.h' file not found
#include "ui/CLS/CLSSIS3820ScalerChannelViewWithDarkCurrent.h"

in

sub-BioXASMainAcquaman_internal-pro-make_default

**Pull request merged**